livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py

```diff
@@ -105,8 +105,8 @@ async def list_voices(self) -> List[Voice]:
 
     async def synthesize(
         self,
-        *,
         text: str,
+        **_,
```

Comment:
What is the reason for **_, here?

Reply:
This was to match signature with base class. But we can take that discussion to the other thread